clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -364,7 +364,7 @@ int main() {
     boo();
     goo();
     zoo();
-    woo();
+    loop_attrs_compatibility();
```

Comment:
Can't stop laughing on this test: "foo, boo, goo. zoo... loop_attrs_compatibility" :)